doc/source/analytics/analytics.md

```diff
@@ -36,20 +36,20 @@ We recommend to configure Prometheus using [Prometheus Operator](https://github.
 The [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) stack configuration can be easily installed using the [Bitnami Helm Charts](https://github.com/bitnami/charts/tree/master/bitnami/kube-prometheus/)
 
 ```bash
-kubectl create namespace seldon-system
+kubectl create namespace seldon-monitoring
 
-helm upgrade --install prometheus kube-prometheus \
-    --version 6.8.3 \
+helm upgrade --install seldon-monitoring kube-prometheus \
+    --version 6.9.5 \
 	--set fullnameOverride=seldon-monitoring \
-    --namespace seldon-system \
+    --namespace seldon-monitoring \
```

Comment:
maybe we can just update to "prometheus" ns given it's no longer anything seldon specific

Reply:
Yes, the ansible in here needed a cleanup for quite some time. And on prometheus it is just a change of namespace in which example/docs installs it.

Reply:
Currently we do suggest namespaces like `seldon-logs` so it would align for consistency so can go ahead